app.js

```diff
@@ -162,3 +162,9 @@ TeamValidator.PM.spawn();
  *********************************************************/
 
 require('./repl').start('app', cmd => eval(cmd));
+
+/*********************************************************
+ * Load the GitHub plugin
+ *********************************************************/
+
+Object.assign(Chat.commands, require('./chat-plugins/github').commands);
```

Comment:
The point of having it in `chat-plugins` was so that this part is unnecessary. We may need to add some sort of API to facilitate hotpatching, though.